lib/media/track/localmediatrack.js

```diff
@@ -33,23 +33,40 @@ function mixinLocalMediaTrack(AudioOrVideoTrack) {
         },
         id: {
           enumerable: true,
-          value: mediaStreamTrack.id
+          value: mediaTrackSender.id
         },
         isEnabled: {
           enumerable: true,
           get() {
-            return mediaStreamTrack.enabled;
+            return mediaTrackSender.enabled;
           }
         },
         isStopped: {
           enumerable: true,
           get() {
-            return mediaStreamTrack.readyState === 'ended';
+            return mediaTrackSender.readyState === 'ended';
           }
         }
       });
     }
 
+    /**
+     * @private
+     * replaces underlying track and
+     * returns a promise.
+     */
+    _replaceTrack(track) {
```

Comment:
@makarandp0 Nitpick:
```suggestion
    _setMediaStreamTrack(mediaStreamTrack) {
```